Doc/library/traceback.rst

```diff
@@ -237,20 +249,23 @@ The module defines the following functions:
 
    .. versionadded:: 3.5
 
-The module also defines the following classes:
 
 :class:`!TracebackException` Objects
 ------------------------------------
 
 .. versionadded:: 3.5
 
-:class:`!TracebackException` objects are created from actual exceptions to
-capture data for later printing in a lightweight fashion.
+:class:`TracebackException` objects are created from actual exceptions to
```

Comment:
This link just takes you down to the next paragraph. I don't think it's useful, and too many links in a page just distract the reader. I would continue to suppress it:

```suggestion
:class:`!TracebackException` objects are created from actual exceptions to
```

On the other hand, when you mention "traceback objects" and "frame objects" later on in this paragraph, you could easily add links to [frame objects](https://docs.python.org/3/reference/datamodel.html#frame-objects) and [traceback objects](https://docs.python.org/3/reference/datamodel.html#traceback-objects) in the data model (it would be `` :ref:`frame-objects` `` and `` :ref:`traceback-objects` ``, respectively).